### DISCUSSION TITLE
Page performance analytics

### DISCUSSION THREAD

**Discussion by NathanC** [plausible/analytics#2045] [Anything Else]
It'd be really cool if there was a way to measure page load times. I'm using a React app, and after navigating to a new page (e.g., a blog post) there's a delay while loading the post from the backend. It'd be great to see the average load time for resources on the page, as well as outliers.

I think the answer's no, but is there any way to do this with the current features?

**Comment by metmarkosaric** [comment]
Thanks for the feedback @NathanC! We're focused on web analytics at the moment but might extend into web performance measurement in the future. There's more here #161
